packages/font-glyphs/src/symbol/math/large-operators.ptl

```diff
@@ -72,11 +72,25 @@ glyph-block Symbol-Math-Large-Operators : for-width-kinds WideWidth1
 
 	create-glyph [MangleName 'Top'] [MangleUnicode 0x27D9] : glyph-proc
 		set-width df.width
-		include : HBar.t df.leftSB df.rightSB BgOpTop OperatorStroke
-		include : VBar.m df.middle BgOpTop BgOpBot OperatorStroke
+		include : HBar.t df.leftSB df.rightSB BgTkTop OperatorStroke
+		include : VBar.m df.middle BgOpTop BgTkBot OperatorStroke
```

Comment:
I'll suppose this BgOpTop should be changed to BgTkTop as well?

Reply:
good catch